Resources/Prototypes/tags.yml

```diff
@@ -310,6 +310,9 @@
 - type: Tag
   id: StringInstrument
 
+- type: Tag
+  id: SyndicateSegwayKeys
+
```

Comment:
there's a key somewhere that is tagged with every possible key as a debug spawn item, add this tag there as well